src/tools/shapes/CircleCreator.ts

```diff
@@ -81,12 +81,10 @@ export default class CircleCreator implements Shape {
     layer.batchDraw()
   }
 
-  getGroup = (): Konva.Group => {
-    return this.group
-  }
-
   // eslint-disable-next-line
   [key: string]: any;
+
+  getGroup = (): Konva.Group => this.group
```

Comment:
Could we do anything about the repetitive code with getGroup? 
Could each shape extend a class that has this and other common methods in it? So each shape then inherits/extends these method and we only have one place to update?